StripeUICore/StripeUICore/Source/Categories/UIColor+StripeUICore.swift

```diff
@@ -215,4 +230,30 @@ private extension UIColor {
         })
     }
 
+    /// Applies a mask over a base color
+    ///
+    /// - Parameters:
+    ///   - overlayColor: The color to overlay
+    ///   - baseColor: The base color
+    /// - Returns: The computed color of the overlay color on the base color.
+    func overlayColor(overlayColor: UIColor, baseColor: UIColor) -> UIColor {
```

Comment:
Is Android computing the overlay color the same?

Reply:
I'm not sure, I'll ask